android/src/main/java/com/onegini/mobile/sdk/reactnative/RNOneginiSdk.kt

```diff
@@ -276,8 +276,22 @@ class RNOneginiSdk(reactContext: ReactApplicationContext) : ReactContextBaseJava
 
     @ReactMethod
     fun cancelRegistration(promise: Promise) {
-        oneginiSDK.registrationRequestHandler.cancelRegistration()
-        promise.resolve(null)
+        var canceled = oneginiSDK.registrationRequestHandler.cancelRegistration()
+        canceled = oneginiSDK.createPinRequestHandler.pinCancelled() || canceled
+        canceled = cancelCustomRegistrations() || canceled
+        if (canceled) {
+            promise.resolve(null)
+        } else {
+            promise.reject(OneginiWrapperErrors.REGISTRATION_NOT_IN_PROGRESS.code, OneginiWrapperErrors.REGISTRATION_NOT_IN_PROGRESS.message)
+        }
+    }
+
+    private fun cancelCustomRegistrations(): Boolean {
```

Comment:
It's a good practice to name methods which return `Boolean`s with prefix `is` `should` `has` etc. Method name should be a question, and method result should be an answer.